Jenkinsfile

```diff
@@ -0,0 +1,72 @@
+#!groovy
+//properties([pipelineTriggers([cron('H H/6 * * *')])])
+node('slave1-dev-jenkins') {
```

Comment:
I am not going to even pretend I understand this file. What's this for? Is it related to this PR?